bot/internal/bot/docpaths.go

```diff
@@ -0,0 +1,122 @@
+package bot
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/gravitational/shared-workflows/bot/internal/github"
+	"github.com/gravitational/trace"
+)
+
+// DocsConfig represents the structure of the config.json file found in the docs
+// directory of a Teleport repo. We only use Redirects, so other fields have the
+// "any" type.
+type DocsConfig struct {
+	Navigation any
+	Variables  any
+	Redirects  RedirectConfig
+}
+
+type RedirectConfig []Redirect
+
+type Redirect struct {
```

Comment:
This will be referred to as `bot.Redirect` - which doesn't make it super obvious that it's related to docs.

I would either have a `docs` package so this can be named `docs.Redirect` or rename this to something like `DocsRedirect`.